src/actions/profile-view.js

```diff
@@ -845,7 +841,7 @@ export function changeExpandedCallNodes(
 
 export function changeSelectedMarker(
   threadIndex: ThreadIndex,
-  selectedMarker: IndexIntoRawMarkerTable | -1
```

Comment:
This was a blatant lie, as this was an index to the PreviewFilteredMarkers array.

Reply:
Yikes.